newrelic-agent/src/main/java/com/newrelic/api/agent/NewRelicApiImplementation.java

```diff
@@ -127,6 +131,7 @@ public void noticeError(String message, Map<String, ?> params, boolean expected)
         try {
             ServiceFactory.getRPMService().getErrorService().reportError(message, filterErrorAtts(params, attributeSender), expected);
             MetricNames.recordApiSupportabilityMetric(MetricNames.SUPPORTABILITY_API_NOTICE_ERROR);
+
             if (expected) {
                 MetricNames.recordApiSupportabilityMetric(MetricNames.SUPPORTABILITY_API_EXPECTED_ERROR_API_MESSAGE);
             }
```

Comment:
There also aren't good options for this supportability metric either.  If this was set by config, then this metric isn't really accurate. I don't really see a great way to clean that up.